sdk/core/azure-core/src/test/java/com/azure/core/http/rest/PercentEscaperTests.java

```diff
@@ -3,6 +3,7 @@
 
 package com.azure.core.http.rest;
 
+import com.azure.core.implementation.http.rest.PercentEscaper;
```

Comment:
instead of doing this we should move all these tests to `com.azure.core.implementation.http.rest` , like we did with classes under test in main source set.